lib/spack/spack/spec.py

```diff
@@ -2258,6 +2258,8 @@ def replace_hash(self):
 
         self._dup(self.lookup_hash())
 
+    # TODO/RepoSplit: Replace the docstring namespace to new package repo to
+    # TODO/RepoSplit:   avoid confusion since no longer "built in".
```

Comment:
I don't think this needs to change -- `builtin` will still be a valid example of a namespace, even if it's not one we use by default anymore.